source/fundamentals/enterprise-auth.txt

```diff
@@ -150,12 +153,42 @@ You may need to specify one or more of the following additional
       - :java-docs:`CANONICALIZE_HOST_NAME_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#CANONICALIZE_HOST_NAME_KEY>`
       - :java-docs:`JAVA_SUBJECT_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_KEY>`
       - :java-docs:`JAVA_SASL_CLIENT_PROPERTIES_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SASL_CLIENT_PROPERTIES_KEY>`
+      - :java-docs:`JAVA_SUBJECT_PROVIDER_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_PROVIDER_KEY>`
 
       Your code to instantiate a ``MongoClient`` using GSSAPI and additional
       properties might resemble the following:
 
       .. include:: /includes/fundamentals/code-snippets/auth-credentials-gssapi-properties.rst
 
+
+The Java Driver will by default cache Kerberos tickets by ``MongoClient`` instance.
```

Comment:
Always try to use present tense instead of future:
https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/

E.g. The Java driver caches Kerberos tickets by MongoClient instance by default.

Reply:
Thank you for the note Chris. The double use of 'by' in the phrase, "by MongoClient instance by default" was a bit hard for me to understand on first reading. Are you ok with this rewrite:

"By default, the Java Driver caches Kerberos tickets by ``MongoClient`` instance."

Reply:
Me and Chris discussed in check-in, and got LGTM.